src/azure-cli/azure/cli/command_modules/acs/_breaking_change.py

```diff
@@ -0,0 +1,9 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------
+
+from azure.cli.core.breaking_change import register_default_value_breaking_change
+
+register_default_value_breaking_change('aks create', '--node-vm-size', 'Standard_DS2_V2 (Linux), Standard_DS2_V3 (Windows)', 'Dynamically Selected By Azure')
+# The default value of `--node-vm-size` will be changed to `Dynamically Selected By Azure` from `Standard_DS2_V2 (Linux), Standard_DS2_V3 (Windows)` in next breaking change release(2.xx.0).
```

Comment:
According to the release plan `2.xx.0` should be `2.73.0`

https://github.com/Azure/azure-cli/milestone/155